samples/05_content_publishers/pdf_table_to_pdf_map.ipynb

```diff
@@ -0,0 +1,1459 @@
+{
```

Comment:
<ol><li>no apostrophe in "panda's data frame". -&gt; Pandas <code>DataFrame</code></li><li>Missing space in "machinebefore" words</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>